### PR TITLE
FEWD-9678 3.1.6 relnotes

### PR DESCRIPTION
**Pull request by ElliotFrancisHunter**
Release notes for 3.1.6 on all mobile platforms.

### CODE REVIEW

File: modules/java/partials/release-notes/couchbase-mobile-java-release-note.3.0.5.adoc [hunk 1]
```diff
@@ -1,8 +1,3 @@
-
-
-
-
-
 [#maint-3-0-5]
```
Comment (osfameron) [inline modules/java/partials/release-notes/couchbase-mobile-java-release-note.3.0.5.adoc]: Might be even better to delete this file, as it's 3.0 and hidden in the 3.1 release branch

**Review by osfameron — APPROVED** [review]
lgtm

**Review by djpongh — COMMENTED** [review]
All platforms should have the LiteCore issues listed:
CBL-5127, CBL-5254, CBL-5259, CBL-5448
LiteCore is common library that all platforms use.

File: modules/ROOT/pages/_partials/supported-versions.adoc [hunk 2]
```diff
@@ -43,6 +38,26 @@ ifdef::is-fullpage[== Officially Supported Versions]
 
 Couchbase Lite for C is available on the platforms shown in the tables below.
 
+[IMPORTANT]
```
Comment (osfameron) [inline modules/ROOT/pages/_partials/supported-versions.adoc:41]: This markup looks great, don't think we lose anything by removing the indirection 👍 
Assume you've tested and the rendered output looks good.

**Review by osfameron — APPROVED** [review]
👍

File: modules/ROOT/pages/_partials/supported-versions.adoc (hunk 2 above)
Comment (ElliotFrancisHunter) [reply modules/ROOT/pages/_partials/supported-versions.adoc:41]: Yep it's exactly the same 👍🏼

**Pull request merged**